### ISSUE TITLE
Checkbox for sharing when adding a printer

### ISSUE THREAD

**Issue by lazyDesertMan** [OpenPrinting/system-config-printer#375]
In the printer add menu there is no checkbox or any other way to specify whether to use shared access or not when adding a device. The device is set to the default value defined in CUPS after adding.

Since in some systems the default value is not overridden, and CUPS sets shared access by default, this leads to the fact that in networks with a large number of devices (for example, in schools), the user sees a very large list of devices that were not initially intended to be shared. After adding, shared access can be disabled in the context menu, but users do not always do this.

Perhaps it is worth adding new line, for example, in the "Describe Printer" section, containing a checkbox that allows users to specify whether to use shared access or not for the device being added?

**Comment by zdohnal** [comment]
Yes, that would be nice - unfortunately I don't have much time for this project. but if someone sends the PR, I will review and merge it.

**Comment by lazyDesertMan** [comment]
https://github.com/OpenPrinting/system-config-printer/pull/376

**Comment by alexpevzner** [comment]
Hi @zdohnal,

@lazyDesertMan has sent a PR. I've reviewed it, as for me, it looks good. Can we proceed now?

**Comment by zdohnal** [comment]
@alexpevzner thx!
